tests/modules/router.test.ts

```diff
@@ -373,6 +373,13 @@ describe('Testing HTTP methods', () => {
 
     router.all('/', () => void 0)
 
+    expect(router.middleware[0].type).toBe('route')
+  })
+  it('app.get with array of paths should set GET as HTTP method', () => {
+    const router = new Router()
+
+    router.get(['/', '/test'], () => void 0)
+
     expect(router.middleware[0].type).toBe('route')
```

Comment:
I think it'd make sense to assert `middleware.length` instead

Reply:
Is this how the length is checked? 
`expect(router.middleware.length).toHaveLength(2)`

Reply:
Fixed in commit ad67943ae61c9e428f7bf41ffd424ca2b2d94ddd